### PR TITLE
Plot of incident growth

### PR DESCRIPTION
**Pull request by lmcnulty**
Per #1156. [Deploy Preview](https://deploy-preview-1164--staging-aiid.netlify.app/summaries/incidentsOverTime/)

Current status:

image

<details>
<summary>Old Image</summary>

</details>

### CODE REVIEW

File: site/gatsby-site/src/pages/summaries/incidentsOverTime.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      const earliestReport = reports.sort((a, b) => a.date_submitted - b.date_submitted)[0];
+
+      return {
+        ...incident,
+        date_submitted: earliestReport.date_submitted,
+      };
+    })
+    .sort((a, b) => Date.parse(a.date_submitted) - Date.parse(b.date_submitted));
+
+  console.log(`incidentsByDate`, incidentsByDate);
```
Comment (smcgregor) [inline site/gatsby-site/src/pages/summaries/incidentsOverTime.js:51]: Console (just in case you missed it, I see this is WIP)

Comment (smcgregor) [reply site/gatsby-site/src/pages/summaries/incidentsOverTime.js:51]: NM, it is instrumented at the moment and likely just because it is draft.

**Comment by smcgregor** [comment]
It looks like the axes are being truncated?

**Comment by lmcnulty** [comment]
@smcgregor I set the x-axis to start at March 28th, 2022, which was the date of the RAIC founding report. I'll probably make this configurable.

**Comment by smcgregor** [comment]
Looks good to me. I think we will soon need a button in the left menu for summaries, with pages like this linked from the summaries page.

**Review by smcgregor — APPROVED** [review]

**Pull request merged**